demos/helloasyncio/helloasyncio.py

```diff
@@ -0,0 +1,30 @@
+import tornado.httpserver
+import tornado.ioloop
+import tornado.options
+import tornado.web
+
+from tornado.options import define, options
+
+from tornado.ioloop import IOLoop
+IOLoop.configure('tornado.platform.asyncio.AsyncIOLoop')
```

Comment:
I agree we need better documentation of asyncio usage, but I don't think copying the hello world example with one line added (and no commentary) is the way to do it.